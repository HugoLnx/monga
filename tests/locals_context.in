int numGlobal;
char cGlobal;
int main() {
  int num;
  char c;
  {
    int num2;
    char c2;

    numGlobal = 29;
    num = 13;
    num2 = 17;
    cGlobal = 'Z';
    c = 'X';
    c2 = 'K';

    printf("numGlobal: %d\n", numGlobal);
    printf("num: %d\n", num);
    printf("num2: %d\n", num2);
    printf("cGlobal: %c\n", cGlobal);
    printf("c: %c\n", c);
    printf("c2: %c", c2);
  }
}
