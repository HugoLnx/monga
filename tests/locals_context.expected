numGlobal: 29
num: 13
num2: 17
cGlobal: Z
c: X
c2: K
